src/renderer/extensions/vueNodes/VideoPreview.vue

```diff
@@ -207,6 +220,16 @@ const handleMouseLeave = () => {
   isHovered.value = false
 }
 
+const handleFocusIn = () => {
+  isFocused.value = true
```

Comment:
Nit: It really feels like we're using JavaScript to do what the DOM does well otherwise (hovered/focused state management)